doc/source/istanbul.md

```diff
@@ -0,0 +1,32 @@
+# 🏘️ Istanbul
+
+Istanbul, even though some still prefer to call it Constantinople, is the name of this city since 1930 when its name was officially changed to Istanbul. The word “Istanbul” is derived from Greek phrase "εἰς τὴν Πόλιν" which means “to the city”. Even though the city was called as “Der Saadet” meaning the “gate to prosperity” in Ottoman Turkish, the name Istanbul was preferred as it was used since 17th century.
```

Comment:
Istanbul, even though some still prefer to call it Constantinople, is the name of this city since 1930 when its name was officially changed to Istanbul. The word “Istanbul” is derived from the Greek phrase "εἰς τὴν Πόλιν" which means “to the city”. Even though the city was called “Der Saadet” meaning the “gate to prosperity” in Ottoman Turkish, the name Istanbul was preferred as it was used since the 17th century.